dev-python/numpy/numpy-1.16.2.recipe

```diff
@@ -0,0 +1,133 @@
+SUMMARY="Array processing for numbers, strings, records, and objects"
+DESCRIPTION="NumPy is a general-purpose array-processing package designed to \
+efficiently manipulate large multi-dimensional arrays of arbitrary records \
+without sacrificing too much speed for small multi-dimensional arrays.
+NumPy is built on the Numeric code base and adds features introduced by \
+numarray as well as an extended C-API and the ability to create arrays of \
+arbitrary type which also makes NumPy suitable for interfacing with \
+general-purpose data-base applications."
+HOMEPAGE="http://numpy.scipy.org/"
+COPYRIGHT="2005-2019 Travis E. Oliphant et al."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/numpy/numpy/releases/download/v$portVersion/numpy-$portVersion.tar.gz"
+CHECKSUM_SHA256="8088221e6e27da8d5907729f0bfe798f526836f22cc59ae83a0f867e67416a3e"
+SOURCE_DIR="numpy-$portVersion"
+PATCHES="numpy-$portVersion.patch"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+#	lib:libamd$secondaryArchSuffix
+	lib:libblis$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
+#	lib:libfftw3$secondaryArchSuffix
+	lib:liblapack$secondaryArchSuffix
+	lib:libopenblas$secondaryArchSuffix
+#	lib:libumfpack$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku_devel$secondaryArchSuffix
+#	devel:cblas$secondaryArchSuffix
+#	devel:libamd$secondaryArchSuffix
+	devel:libblis$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+#	devel:libfftw3$secondaryArchSuffix
+	devel:liblapack$secondaryArchSuffix
+	devel:libopenblas$secondaryArchSuffix
+#	devel:libumfpack$secondaryArchSuffix
+	"
+
+PYTHON_PACKAGES=(python python3)
+PYTHON_VERSIONS=(2.7 3.6)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersion=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\n\
+	cmd:f2py\n\
+	cmd:f2py${pythonVersion%.*}\n\
+	cmd:f2py$pythonVersion\n\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku$secondaryArchSuffix\n\
+	cmd:python$pythonVersion\
+	\""
+BUILD_REQUIRES="$BUILD_REQUIRES
+	setuptools_$pythonPackage
+	"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:git
+	cmd:gcc$secondaryArchSuffix
+	cmd:gfortran$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python$pythonVersion
+	"
+done
+
+REPLACES_python="
+	python_numpy
+	"
+
+INSTALL()
+{
+# Check at every update in site.cfg.example!
+cat > site.cfg << EOF
+[ALL]
+libraries = execinfo,lapack,openblas
+library_dirs = /system/`echo ${relativeDevelopLibDir}`
```

Comment:
```suggestion
library_dirs = /system/$relativeDevelopLibDir
```

It'd be shorter to just use the variables directly :P

I'm not a fan of hardcoding, but it seems to be rather complex to patch this up using `findpaths`, so it's fine for this case.

Reply:
Can you do the same with all other variables?